lib/game_teams.rb

```diff
@@ -16,21 +16,21 @@ class GameTeams
              :giveaways,
              :takeaways
 
- def initialize(stats)
-   @game_id                  = stats[:game_id]
-   @team_id                  = stats[:team_id]
-   @hoa                      = stats[:hoa]
-   @won                      = stats[:won]
-   @settled_in               = stats[:settled_in]
-   @head_coach               = stats[:head_coach]
-   @goals                    = stats[:goals]
-   @shots                    = stats[:shots]
-   @hits                     = stats[:hits]
-   @pim                      = stats[:pim]
-   @power_play_opportunities = stats[:powerplayopportunities]
-   @power_play_goals         = stats[:powerplaygoals]
-   @face_off_win_percentage  = stats[:faceoffwinpercentage]
-   @giveaways                = stats[:giveaways]
-   @takeaways                = stats[:takeaways]
+ def initialize(row)
+   @game_id                  = row[0]
+   @team_id                  = row[1]
+   @hoa                      = row[2]
+   @won                      = row[3]
+   @settled_in               = row[4]
+   @head_coach               = row[5]
+   @goals                    = row[6].to_i
+   @shots                    = row[7].to_i
+   @hits                     = row[8].to_i
+   @pim                      = row[9].to_i
+   @power_play_opportunities = row[10].to_i
+   @power_play_goals         = row[11].to_i
+   @face_off_win_percentage  = row[12].to_f
+   @giveaways                = row[13].to_i
+   @takeaways                = row[14].to_i
```

Comment:
Since we are now passing in an array instead of a hash like before, we can now assign instance variables to each value in each row for easier access later.